src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements IntLinkedListInterface {
+
+    //zrobione
+
+    private static final int sizeOfStack = 11;
```

Comment:
constants names should be UPPERCASE!
This constant is related to stackCapacity . Changing only one leads to an error (inconsistency).